server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -2497,6 +2497,38 @@ public void removeInUseLogs(Set<DfsLogger> candidates) {
     candidates.removeAll(referencedLogs);
   }
 
+  public void checkIfMinorCompactionNeededForLogs(List<DfsLogger> closedLogs) {
+
+    // grab this outside of tablet lock.
+    int maxLogs = tableConfiguration.getCount(Property.TABLE_MINC_LOGS_MAX);
+
+    String reason = null;
+    synchronized (this) {
+      if (currentLogs.size() >= maxLogs) {
```

Comment:
Will this effectively subtract 1 from the max logs configuration? If currentLogs.size() is equal to maxLogs, then we'll force a MinC.

Reply:
> If currentLogs.size() is equal to maxLogs, then we'll force a MinC.

Thats correct.  I believe this was the behavior of the code before my change.  The tablet server code used to do the following.  For that case I was trying to maintain the current behavior.

```java
if (tablet.getLogCount() >= maxLogEntriesPerTablet) {
```

Reply:
Ahh, okay. Thanks for the clarification.